front/app/components/ReactionControl/utils.ts

```diff
@@ -17,13 +13,11 @@ export const showIdeaReactions = (
   const likesCount = idea.attributes.likes_count;
   const dislikesCount = idea.attributes.dislikes_count;
   return (
-    participationMethod !== 'voting' &&
-    participationMethod !== 'proposals' &&
```

Comment:
Instead, I've added a check for participationMethod before calling this function. The participationMethod checks should not be in this function.